### PR TITLE
New Plenum component DropdownWithMultiSelect

### PR DESCRIPTION
**Pull request by DerekAgility**
Jira: https://agilitycms.atlassian.net/browse/FE-2385

Added a new Plenum component that acts like a select, but when the dropdown opens, we show a list of checkboxes.

image

### CODE REVIEW

File: .github/workflows/run_unit_test.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
             - name: Set up Node.js
               uses: actions/setup-node@v3
               with:
-                  node-version: "18" # Specify the Node.js version you want
+                  node-version: "20" # Specify the Node.js version you want
```
Comment (DerekAgility) [inline .github/workflows/run_unit_test.yml:22]: bumping these up to match manager app repo

File: stories/molecules/inputs/checkbox/Checkbox.tsx [hunk 2]
```diff
@@ -1,33 +1,35 @@
...
+	hasBorder?: boolean;
 	/** any arbitrary classNames to add to the wrapper */
-	className?: string
-	/** Label ClassName */
-	labelClassName?: string
+	className?: string;
+	/** Truncate label */
+	truncateLabel?: boolean;
+	/** Full width label */
+	fullWidthLabel?: boolean;
```
Comment (DerekAgility) [inline stories/molecules/inputs/checkbox/Checkbox.tsx:32]: just passing these down from the new component to inputlabel

File: stories/molecules/inputs/combobox/ComboBox.tsx [hunk 3]
```diff
@@ -1,47 +1,47 @@
-import React, { useState, useEffect } from "react"
-import { default as cn } from "classnames"
-import { Combobox as HeadlessUICombobox } from "@headlessui/react"
-import InputLabel from "@/stories/molecules/inputs/InputLabel"
-import { DynamicIcon } from "@/stories/atoms/icons"
+import { useState, useEffect } from "react";
```
Comment (DerekAgility) [inline stories/molecules/inputs/combobox/ComboBox.tsx:1]: These are ts fixes that had to be done since we upgraded node

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@agility/plenum-ui",
-	"version": "2.1.18",
+	"version": "2.1.20-rc8",
```
Comment (DakotahGodfrey) [inline package.json:3]: If this is ready to be published, can you update version to 2.1.20?

**Review by DakotahGodfrey — APPROVED** [review]
Approving but can you update the package version?

**Review by DakotahGodfrey — APPROVED** [review]

**Pull request merged**